modules/ROOT/pages/getting-started.adoc

```diff
@@ -74,12 +74,10 @@ include::keycloak/keycloak-setup.adoc[leveloffset=2]
 
 === Supported Environments
 
-In order to perform local development, you will need to have set up a local development environment or IDE. {product-name} supports mobile app development across iOS Native, Android Native, Cordova and Xamarin. For information on how to set up your local development environment for these technologies, please see the links below:
+In order to perform local development, you will need to have set up a local development environment or IDE.
+{product-name} supports mobile app development across iOS Native, Android Native, Cordova and Xamarin.
 
-* link:https://developer.android.com/studio/install[Android, window="_blank"]
-* link:https://developer.apple.com/library/archive/referencelibrary/GettingStarted/DevelopiOSAppsSwift/index.html[iOS, window="_blank"]
-* link:https://cordova.apache.org/#getstarted[Cordova, window="_blank"]
-* link:https://developer.xamarin.com/getting-started/[Xamarin, window="_blank"]
+include::{partialsdir}/generic-local-mobile-dev-envir-reqs.adoc[]
```

Comment:
@finp - any idea why this won't render? I cannot figure it out. I'm quite sure I've used it like other partial imports...can't see what is up.